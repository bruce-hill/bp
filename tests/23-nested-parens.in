foo(one(), two(three()));
foo();
xx()))));
xx(((();
yy)()();
