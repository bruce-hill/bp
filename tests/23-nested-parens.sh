# parens is a pattern matching nested parentheses
# Example: bp -p '"foo" parens' matches "foo()" or "foo(baz(), qux())", but not "foo(()"
bp -p 'id parens `;'
