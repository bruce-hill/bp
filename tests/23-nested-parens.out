foo(one(), two(three()));
foo();
